.github/dependabot.yml

```
@@ -33,3 +33,13 @@ updates:
       - "🟩 priority: low"
       - "🐍 tech: python"
       - dependencies
+      
+  - package-ecosystem: github-actions
```

To keep the file same as the [one in the catalog](https://github.com/WordPress/openverse-catalog/blob/main/.github/dependabot.yml).

```suggestion
  - package-ecosystem: "github-actions"
```